torch_frame/data/dataset.py

```diff
@@ -695,7 +696,7 @@ def col_select(self, cols: ColumnSelectType) -> Dataset:
 
         return dataset
 
-    def get_split(self, split: str) -> Dataset:
+    def get_split(self, split: TrainingStage) -> Dataset:
```

Comment:
I think it's nicer to accept `str` as input here. It's sometimes troublesome to import `TrainingStage`.

Reply:
Maybe we can support both? i.e. `class TrainingStage(str, Enum)`  I feel like `TrainingStage` has its advantage in that no mistypes will occur that causes unexpected behavior, e.g. 'Train' vs 'train'. I feel like this is also the reason we support lots of Enum classes in the codebase

Reply:
Yes, we can support both.

Reply:
`Union[str, TrainingStage]`